core/src/main/java/com/radixdlt/consensus/ConsensusHasher.java

```diff
@@ -83,8 +83,7 @@ public static HashCode toHash(
       outputStream.writeInt(header != null ? 0 : 1); // 4 bytes (Version)
       outputStream.write(opaque.asBytes()); // 32 bytes
       if (header != null) {
-        outputStream.write(header.getAccumulatorState().getAccumulatorHash().asBytes()); // 32 bytes
-        outputStream.writeLong(header.getAccumulatorState().getStateVersion()); // 8 bytes
+        outputStream.writeLong(header.getStateVersion()); // 8 bytes
         // TODO: include the header.getStateHash().asBytes() when we are ready to affect consensus
```

Comment:
_(note from author)_

@dhedey I think we missed this TODO, and...
1. Right now we could probably include all the `LedgerHashes` here, right?
2. AFAIU, this would effectively render the checks inside Rust `prepare()` useless (i.e. impossible to ever trigger on prod), should we keep them as sanity checks anyways?

Reply:
(let's discuss it in a spin-off https://github.com/radixdlt/babylon-node/pull/479)